client/src/locales/translations/en_US.json

```diff
@@ -369,7 +369,7 @@
     "ourIdea": "Our Idea",
     "ourSupporters": "Our Supporters",
     "followUs": "Follow us on social media",
-    "contactUs": "Contact us at: ",
+    "contactUs": "Or contact us at: ",
     "contactEmail": "contact@fightpandemics.com",
```

Comment:
Your fix to the reported issue is good - thanks @ddveloper  ! However, I'm wondering why do we `contactEmail` as a translation item?

Reply:
you're right @joshmorel , the only place uses this is in the AboutUs page showing the exact same characters, I guess we can delete that pair, or put it somewhere under the `constants` folder